sda-commons-starter-example/README.md

```diff
@@ -0,0 +1,25 @@
+# SDA Commons Starter Example
```

Comment:
As a next step, we should think about aligning this example nearer to our template-project and maybe also make an effort in improving the overall documentation, if needed.